### ISSUE TITLE
Windows 7 installation - problem with localhost

### ISSUE THREAD

**Issue by ntatsisk** [opencv/cvat#510]
I am trying to install CVAT in Windows 7 using the docker toolbox. However, I cannot connect to localhost:8080.

The output of `docker ps` is: 
```
$ docker ps
CONTAINER ID        IMAGE                  COMMAND                  CREATED             STATUS              PORTS                              NAMES
8d85700b1550        cvat                   "/usr/bin/supervisord"   36 minutes ago      Up 6 minutes        0.0.0.0:8080->8080/tcp, 8443/tcp   cvat
2585643d7d70        redis:4.0.5-alpine     "docker-entrypoint.s…"   36 minutes ago      Up 6 minutes        6379/tcp                           cvat_redis
5df9df40cbf5        postgres:10.3-alpine   "docker-entrypoint.s…"   36 minutes ago      Up 6 minutes        5432/tcp                           cvat_db
```

and the log from `docker logs cvat`:  [cvat_log.txt](https://github.com/opencv/cvat/files/3313702/cvat_log.txt) 

Thanks!

**Comment by nmanovic** [comment]
Hi @ntatsisk , let's do a couple of simple steps to better understand your problem.

- Run `docker-compose down` command
- Run `docker-compose up -d` command
- Go to your browser and open http://127.0.0.1:8080 (try http://localhost:8080 as well)

If it doesn't work please send me your logs one more time. Did CVAT work previously for you?

**Comment by bsekachev** [comment]
Maybe [this](https://stackoverflow.com/questions/42866013/docker-toolbox-localhost-not-working) answer will help you.

**Comment by ntatsisk** [comment]
@nmanovic @bsekachev Thank you for your replies. I followed that link that @bsekachev shared and, indeed, I managed to fix the use by port forwarding. Initially, I was using `80` as port for host and guest but it only worked when I changed it to `8080`.  It is also important to stop and start the box after the changes.

I share an image with my port setup for other people facing the same issue (the names are in dutch but they are evident). Thank you again!

![port_forwarding](https://user-images.githubusercontent.com/45266491/60084625-27034800-9738-11e9-9981-e32a33c7715e.PNG)
